nzedb/Backfill.php

```diff
@@ -112,21 +115,21 @@ public function __construct(array $options = [])
 			}
 		}
 
-		$this->_compressedHeaders = (Settings::value('..compressedheaders') == 1 ? true : false);
-		$this->_safeBackFillDate = (Settings::value('..safebackfilldate') != '') ?
-			(string)Settings::value('..safebackfilldate') : '2008-08-14';
-		$this->_safePartRepair = (Settings::value('..safepartrepair') == 1 ? 'update' : 'backfill');
-		$this->_disableBackfillGroup = (Settings::value('..disablebackfillgroup') == 1 ? true : false);
+		$this->_compressedHeaders = (int) Settings::value('..compressedheaders') === 1;
+		$this->_safeBackFillDate = Settings::value('..safebackfilldate') !== '' ? (string) Settings::value('..safebackfilldate') : '2008-08-14';
```

Comment:
Isn't this cast to string unnecessary as the method returns strings normally?

Reply:
It was a string originaly, so i did not change that.

Reply:
OK, please remove the space then